config/semi_automated.yml

```diff
@@ -161,314 +169,265 @@ parameters:
-        HotCat:
-            regex: 'باستخدام \[\[ويكيبيديا:المصناف الفوري'
-            link: ويكيبيديا:المصناف الفوري
-            label: المصناف الفوري
         Popups:
             link: ويكيبيديا:المنبثقات
             label: المنبثقات
-        WPCleaner:
-            link: ويكيبيديا:وب_كلينر
-            label: وب_كلينر
         Rater:
             regex: '\[\[(:User:FShbib\/المقيم|ويكيبيديا:المقيم)'
             link: مستخدم:FShbib/المقيم
             talk_namespaces: true
-    commons.wikimedia.org:
-        UploadWizard:
-            regex: '^User created page with UploadWizard$'
-            link: Special:MyLanguage/Commons:Upload Wizard
-            namespaces: [6]
-        VisualFileChange:
-            regex: 'Using \[\[COM:VFC'
-            tag: VisualFileChange
-            link: Special:MyLanguage/Help:VisualFileChange.js
-        AjaxQuickDelete:
-            regex: '^(This category needs discussion|\[\[:.*?\]\] needs discussion|Starting category discussion|Nominating for deletion|\[\[:.*?\]\] has been nominated for deletion|Starting deletion request|Listing \[\[.*?\]\])$|#Autoreport by AjaxQuickDelete|^Updating redirect while processing'
-            link: Special:MyLanguage/Help:Nominate_for_deletion
-        QuickDelete:
-            regex: '^(Marking as possible copyvio because|Notification of possible copyright violation for|\[\[.*?\]\] does not have a source|Please send permission for \[\[|\]\] does not have a license)|^(File has no source|Missing permission)$'
-            link: Special:MyLanguage/Help:QuickDelete
-            namespaces: [3, 6]
-        POTY:
-            regex: '\[\[Help:(?:Gadget-)EnhancedPOTY.js|POTY App\]\]$'
-            link: Help:Gadget-EnhancedPOTY.js
-            namespaces: [4]
-        Flickr2Commons:
-            regex: 'via \[\[(?:Special:MyLanguage\/)?Commons:Flickr2Commons'
-            link: Special:MyLanguage/Commons:Flickr2Commons
-            namespaces: [6]
-        Android Commons:
-            regex: '^Uploaded using Android Commons app$'
-            link: Special:MyLanguage/Commons:Mobile_app
-            namespaces: [6]
-        Commonsist:
-            regex: '^commonist \d\.\d'
-            link: Special:MyLanguage/Commons:Commonist
-            namespaces: [6]
-        RenameLink:
-            regex: '^\(\[\[Help:RenameLink\|Script'
-            link: Special:MyLanguage/Help:RenameLink
-        Geograph2commons:
-            regex: 'Transferred from geograph.co.uk using'
-            link: Special:GoToInterwiki/toollabs:geograph2commons
-            namespaces: [6]
-        SettingsManager:
-            regex: '\[\[MediaWiki:Gadget-SettingsManager.js\|SettingsManager'
-            link: MediaWiki:Gadget-SettingsManager.js
-        RotateLink:
-            regex: '\(\[\[Help:RotateLink\|Script'
-            link: Help:RotateLink
-    de.wikipedia.org:
+        WPCleaner:
+            link: ويكيبيديا:وب_كلينر
+            label: وب_كلينر
+        de.wikipedia.org:
```

Comment:
`de.wikipedia.org:` is indented one tab too many. This is why the test was failing :)

Reply:
Done, but that doesn't appear to have been the only issue...

Reply:
This time it was a fluke with Scrutinizer. I triggered another build and it passed.

I'll try to get to doing a full review soon.